packages/frontend/src/js/app/components/map/map.vue

```diff
@@ -415,6 +443,16 @@ export default {
         };
     },
 
+    beforeDestroy() {
+        // before destroying, in case we are in the dashboard page we register position in the store
+        if (this.displayPopupOnTownClick) {
+            this.$store.commit("setDashboardShantytownsMapSetup", {
+                center: this.map.getCenter(),
+                zoom: this.map.getZoom()
+            });
+        }
+    },
+
```

Comment:
Ce bout de code ne devrait pas être dans le composant map, car il s'agit d'une logique métier propre au tableau de bord.
Le mieux serait d'émettre un événement que le Dashboard peut écouter pour effectivement faire le commit dans le store.

(On parle de "separation of concern", si on veut faire fancy).